src/packer.pkr.hcl

```diff
@@ -149,13 +149,189 @@ build {
+    script           = "src/powershell/install-category.ps1"
+    environment_vars = ["CATEGORY=networking"]
+  }
+
+  provisioner "windows-restart" {
+    # Wait a maximum of 30 minutes for Windows to restart.
+    # The build will fail if the restart process takes longer than 30 minutes.
+    restart_timeout = "30m"
+  }
+
+  provisioner "powershell" {
+    # Install password packages
+    script           = "src/powershell/install-category.ps1"
+    environment_vars = ["CATEGORY=passwords"]
+  }
+
+  provisioner "windows-restart" {
+    # Wait a maximum of 30 minutes for Windows to restart.
+    # The build will fail if the restart process takes longer than 30 minutes.
+    restart_timeout = "30m"
+  }
+
+  provisioner "powershell" {
+    # Install reverse engineering packages
+    script           = "src/powershell/install-category.ps1"
+    environment_vars = ["CATEGORY=reverse-engineering"]
+  }
+
+  provisioner "windows-restart" {
+    # Wait a maximum of 30 minutes for Windows to restart.
+    # The build will fail if the restart process takes longer than 30 minutes.
+    restart_timeout = "30m"
+  }
+
+  provisioner "powershell" {
+    # Install utility packages
+    script           = "src/powershell/install-category.ps1"
+    environment_vars = ["CATEGORY=utilities"]
+  }
+
+  provisioner "windows-restart" {
+    # Wait a maximum of 30 minutes for Windows to restart.
+    # The build will fail if the restart process takes longer than 30 minutes.
+    restart_timeout = "30m"
+  }
+
+  provisioner "powershell" {
+    # Install vulnerability analysis packages
+    script           = "src/powershell/install-category.ps1"
+    environment_vars = ["CATEGORY=vulnerability-analysis"]
+  }
+
+  provisioner "windows-restart" {
+    # Wait a maximum of 30 minutes for Windows to restart.
+    # The build will fail if the restart process takes longer than 30 minutes.
+    restart_timeout = "30m"
+  }
+
+  provisioner "powershell" {
+    # Install web application packages
+    script           = "src/powershell/install-category.ps1"
+    environment_vars = ["CATEGORY=web-applications"]
+  }
+
+  provisioner "windows-restart" {
+    # Wait a maximum of 30 minutes for Windows to restart.
+    # The build will fail if the restart process takes longer than 30 minutes.
+    restart_timeout = "30m"
+  }
+
+  provisioner "powershell" {
+    # Install wordlist packages
+    script           = "src/powershell/install-category.ps1"
+    environment_vars = ["CATEGORY=wordlists"]
+  }
+
+  provisioner "windows-restart" {
+    # Wait a maximum of 30 minutes for Windows to restart.
+    # The build will fail if the restart process takes longer than 30 minutes.
+    restart_timeout = "30m"
+  }
+
+  provisioner "powershell" {
+    inline = [
+      # Print information about disk size and usage
+      "wmic logicaldisk get size,freespace,caption",
+      # List each installed chocolatey package
+      "choco list --local-only"
+    ]
   }
```

Comment:
Please make sure you remove this debugging provisioner.

Reply:
@mcdonnnj resolved with [08656a9](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/08656a9670d550fac7c27ecae39dc3a728d5013c)